modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C

```diff
@@ -45,7 +47,9 @@ INSFVRhieChowInterpolator::validParams()
   ExecFlagEnum & exec_enum = params.set<ExecFlagEnum>("execute_on", true);
   exec_enum.addAvailableFlags(EXEC_PRE_KERNELS);
   exec_enum = {EXEC_PRE_KERNELS};
```

Comment:
It looks like you're setting the default to include initial within the action. If you want that, let's just make it the default here so that users running with the action and with a full input file get the same default behavior